common/src/main/java/org/opensearch/sql/common/setting/Settings.java

```diff
@@ -41,6 +41,7 @@ public enum Key {
     QUERY_SIZE_LIMIT("plugins.query.size_limit"),
     ENCYRPTION_MASTER_KEY("plugins.query.datasources.encryption.masterkey"),
     DATASOURCES_URI_ALLOWHOSTS("plugins.query.datasources.uri.allowhosts"),
+    IGNORE_UNSUPPORTED_PAGINATION("plugins.query.ignore_unsupported_pagination"),
```

Comment:
does this only affect sql request?  Maybe `plugins.sql.ignore_unsupported_pagination` or `plugins.sql.cursor.ignore_unsupported`?

Reply:
It affects all pagination requests, but we support pagination in SQL only for now.
I can easily rename the setting.